src/guides/v2.2/release-notes/release-notes-2-2-11-commerce.md

```diff
@@ -0,0 +1,173 @@
+---
+group: release-notes
+title: Magento Commerce 2.2.11 Release Notes
+
+---
+
+**This release (Magento 2.2.11) marks the final supported software release for Magento version 2.2. Magento 2.2 will no longer receive security updates or product quality fixes now that its support window has expired**.
+
+Magento Commerce 2.2.11 offers platform upgrades and substantial security changes. This release includes 29 functional fixes and enhancements to the core product and several security enhancements.
+
+Magento 2.2.11 has not been tested with PHP 7.1. PHP 7.1 reached EOL (End of Life) on December 1, 2019. We recommend updating your deployment to a supported version of PHP.
+
+### Security enhancements
+
+Twenty-three security enhancements that help close cross-site scripting (XSS) and remote code execution (RCE) vulnerabilities as well as other security issues. No confirmed attacks related to these issues have occurred to date. However, certain vulnerabilities can potentially be exploited to access customer information or take over administrator sessions. Most of these issues require that an attacker first obtains access to the Admin. As a result, we remind you to take all necessary steps to protect your Admin, including but not limited to these efforts: IP whitelisting, two-factor authentication, use of a VPN, the use of a unique location rather than /admin, and good password hygiene.
+
+With this quarterly release, we’ve changed how we describe these security issues.  Individual issues are no longer described in the Magento Security Center. Instead, these issues are documented in an Adobe Security bulletin. Please see [Security updates available for Magento | APSB20-02](https://helpx.adobe.com/security/products/magento/apsb20-02.html) for more information.
```

Comment:
There are some spacing problems in this paragraph. Also, there seems to be some strange characters in the link text that causes formatting problems.
 
```suggestion
With this quarterly release, we’ve changed how we describe these security issues. Individual issues are no longer described in the Magento Security Center. Instead, these issues are documented in an Adobe Security bulletin. Please see [Security updates available for Magento | APSB20-02](https://helpx.adobe.com/security/products/magento/apsb20-02.html)  for more information.
```

Reply:
Fixed in #6470